engine/src/main.cc

```diff
@@ -154,6 +159,10 @@ int main(int argc, char* argv[]) {
         case 'D':  // Diagnostic.
           diagnose = true;
           break;
+        case 'c':
+          if (optarg)
+            extended_conf_path.insert(optarg);
```

Comment:
Could we call that variable extended_conf_files as it is a file? Or I missed something...